templates/rhdm70-full.yaml

```diff
@@ -13,160 +13,168 @@ labels:
   description: The name for the application.
   name: APPLICATION_NAME
   value: myapp
   required: true
 - displayName: EAP Admin User
-  description: EAP administrator username
+  description: EAP administrator user name. Use this user account if you need to log on to the JBoss EAP console.
   name: ADMIN_USERNAME
   value: eapadmin
   required: false
 - displayName: EAP Admin Password
-  description: EAP administrator password
+  description: EAP administrator password.
   name: ADMIN_PASSWORD
   from: "[a-zA-Z]{6}[0-9]{1}!"
   generate: expression
   required: false
-- displayName: KIE Admin User
-  description: KIE administrator username
+- displayName: KIE Admin User. Use this user account to log on to Decision Central. 
+  description: KIE administrator user name.
   name: KIE_ADMIN_USER
   value: adminUser
   required: false
 - displayName: KIE Admin Password
-  description: KIE administrator password
+  description: KIE administrator password.
   name: KIE_ADMIN_PWD
   from: "[a-zA-Z]{6}[0-9]{1}!"
   generate: expression
   required: false
-- displayName: KIE Server Controller User
-  description: KIE server controller username (Sets the org.kie.server.controller.user system property)
+- displayName: KIE Server Controller User. 
+  description: KIE server controller user name. The Decision Server uses this user account to log on to 
+    Decision Central. (Sets the org.kie.server.controller.user system property).
   name: KIE_SERVER_CONTROLLER_USER
   value: controllerUser
   required: false
 - displayName: KIE Server Controller Password
-  description: KIE server controller password (Sets the org.kie.server.controller.pwd system property)
+  description: KIE server controller password (sets the org.kie.server.controller.pwd system property).
   name: KIE_SERVER_CONTROLLER_PWD
   from: "[a-zA-Z]{6}[0-9]{1}!"
   generate: expression
   required: false
-- displayName: KIE Server User
-  description: KIE execution server username (Sets the org.kie.server.user system property)
+- displayName: KIE Server User. Use this user account for API calls to the Decision Server. 
+  description: KIE execution server user name (sets the org.kie.server.user system property).
   name: KIE_SERVER_USER
   value: executionUser
   required: false
 - displayName: KIE Server Password
-  description: KIE execution server password (Sets the org.kie.server.pwd system property)
+  description: KIE execution server password (sets the org.kie.server.pwd system property).
   name: KIE_SERVER_PWD
   from: "[a-zA-Z]{6}[0-9]{1}!"
   generate: expression
   required: false
 - displayName: KIE Server Bypass Auth User
-  description: KIE execution server bypass auth user (Sets the org.kie.server.bypass.auth.user system property)
+  description: KIE execution server bypass auth user. If this parameter is set to true, the Decision Server accepts
+   API calls without user account information. (Sets the org.kie.server.bypass.auth.user system property).
```

Comment:
I think 'without user account authentication' describe this env. variable better.